.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,27 @@
-<!-- YOU HAVE TO MODIFY THIS TEXT TO FIT YOUR PR. OTHERWISE, YOUR PR WILL BE CLOSED WITHOUT FURTHER COMMENT. -->
-
-Describe the changes you have made here: what, why, ...
+<!-- YOU HAVE TO MODIFY THE TEXT BELOW TO FIT YOUR PR. OTHERWISE, YOUR PR WILL BE CLOSED WITHOUT FURTHER COMMENT. -->
 
 <!-- LINK THE ISSUE WITH THE "Closes" KEYWORD -->
-<!-- Example: Closes (link) OR Closes #xyz -->
+<!-- Example: Closes (link) OR Closes #12345 -->
+
+Closes _____
+Describe the changes you have made here: what, where, why, ...
+If your work is not yet complete, please open a draft pull request. In that case, outline your intended next steps. Do you need feedback? Will you continue in parallel? ...
 
 ### Mandatory checks
 
 <!--
-- Go through the list below. Please don't remove any items.
-- [x] done; [ ] not done / not applicable
+Go throgh the checklist below. It is mandatory, even for a draft pull request.
+
+Keep ALL the items. Mark them as follows: 
+[x] done 
+[ ] not done 
+[/] not applicable
```

Comment:
`[/]` is not supported by GitHub:

![image](https://github.com/user-attachments/assets/7d010224-087b-4693-a5ca-7284956a87d8)

But I find it helpful to separate n/a from not done ^^